measures/ResStockArgumentsPostHPXML/measure.rb

```diff
@@ -945,16 +948,60 @@ def set_electric_panel(runner, hpxml_bldg_existing, hpxml_bldg, args)
+    args[:electric_panel_load_bathroom_fans_new_load] = (hpxml_bldg_existing.ventilation_fans.size { |vf| vf.fan_location == HPXML::LocationBath } == 0) && (hpxml_bldg.ventilation_fans.count { |vf| vf.fan_location == HPXML::LocationBath } > 0)
+    args[:electric_panel_load_electric_water_heater_new_load] = (hpxml_bldg_existing.water_heating_systems.count { |whs| whs.fuel_type == HPXML::FuelTypeElectricity } == 0) && (hpxml_bldg.water_heating_systems.count { |whs| whs.fuel_type == HPXML::FuelTypeElectricity } > 0)
+    args[:electric_panel_load_electric_clothes_dryer_new_load] = (hpxml_bldg_existing.clothes_dryers.count { |cd| cd.fuel_type == HPXML::FuelTypeElectricity } == 0) && (hpxml_bldg.clothes_dryers.count { |cd| cd.fuel_type == HPXML::FuelTypeElectricity } > 0)
+    args[:electric_panel_load_dishwasher_new_load] = (hpxml_bldg_existing.dishwashers.size == 0) && (hpxml_bldg.dishwashers.size > 0)
+    args[:electric_panel_load_electric_cooking_range_new_load] = (hpxml_bldg_existing.cooking_ranges.count { |cr| cr.fuel_type == HPXML::FuelTypeElectricity } == 0) && (hpxml_bldg.cooking_ranges.count { |cr| cr.fuel_type == HPXML::FuelTypeElectricity } > 0)
+    args[:electric_panel_load_misc_plug_loads_well_pump_new_load] = (hpxml_bldg_existing.plug_loads.count { |pl| pl.plug_load_type == HPXML::PlugLoadTypeWellPump } == 0) && (hpxml_bldg.plug_loads.count { |pl| pl.plug_load_type == HPXML::PlugLoadTypeWellPump } > 0)
+    args[:electric_panel_load_misc_plug_loads_vehicle_new_load] = (hpxml_bldg_existing.plug_loads.count { |pl| pl.plug_load_type == HPXML::PlugLoadTypeElectricVehicleCharging } == 0) && (hpxml_bldg.plug_loads.count { |pl| pl.plug_load_type == HPXML::PlugLoadTypeElectricVehicleCharging } > 0)
+    args[:electric_panel_load_pool_pump_new_load] = (hpxml_bldg_existing.pools.size == 0) && (hpxml_bldg.pools.size > 0)
+    args[:electric_panel_load_electric_pool_heater_new_load] = (hpxml_bldg_existing.pools.count { |pl| [HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(pl.heater_type) } == 0) && (hpxml_bldg.pools.count { |pl| [HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(pl.heater_type) } > 0)
+    args[:electric_panel_load_permanent_spa_pump_new_load] = (hpxml_bldg_existing.permanent_spas.size == 0) && (hpxml_bldg.permanent_spas.size > 0)
+    args[:electric_panel_load_electric_permanent_spa_heater_new_load] = (hpxml_bldg_existing.permanent_spas.count { |ps| [HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(ps.heater_type) } == 0) && (hpxml_bldg.permanent_spas.count { |ps| [HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(ps.heater_type) } > 0)
+    args[:electric_panel_load_other_new_load] = (electric_panel_load_other_power_rating > hpxml_bldg_existing.electric_panels[0].service_feeders.select { |sf| sf.type == HPXML::ElectricPanelLoadTypeOther }.map { |sf| sf.power }.sum)
```

Comment:
Another thought here. "Other" loads account for microwaves, garbage disposals, garage doors, and any other power passed in from the lookup. If, say, the upgraded building increased this load relative the existing building then `new_load` would be set to true. But shouldn't only the _new load_ be assigned to true?

Maybe this is out of scope, and should be addressed as a future item.